WordPress/WordPressTest/Models/WPAccountTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+class WPAccountTests: XCTestCase
+{
+    private let accountEntityName = "Account"
+    private let oneHundredYearsInMilliseconds: NSTimeInterval = 1000 * 60 * 60 * 24 * 7 * 52 * 100
+
+    private var account: WPAccount!
+    
+    override func setUp() {
+        super.setUp()
+        
+        let context = TestContextManager.sharedInstance().mainContext
```

Comment:
When writing unit tests, always try to make sure that no external interactions are possible.  This is not always possible but there's a small improvement opportunity here.

By working on the main context, any other thread calling `[context save]` on the main context will also save your test account into the persistent store.  We want to avoid this if possible.

```swift
    private var context: NSManagedObjectContext!
    private var account: WPAccount!
    
    override func setUp() {
        super.setUp()
        
        context = NSManagedObjectContext(concurrencyType: .PrivateQueueConcurrencyType)
        context.parentContext = TestContextManager.sharedInstance().mainContext
        
        account = NSEntityDescription.insertNewObjectForEntityForName(accountEntityName, inManagedObjectContext: context) as! WPAccount
    }
    
    override func tearDown() {
        account = nil
        context.rollback()
        
        super.tearDown()
    }
```

By using a newly created private context, you make sure that no-one can save the changes on it.  Also, no-one can make changes to it that can impact your testing code.